flutter_local_notifications/example/lib/main.dart

```diff
@@ -1178,6 +1185,84 @@ class _HomePageState extends State<HomePage> {
           );
         });
   }
+
+  Future<void> _getActiveNotifications() async {
+    Widget activeNotificationsDialogContent =
+        await _getActiveNotificationsDialogContent();
+    await showDialog<void>(
+        context: context,
+        builder: (BuildContext context) {
+          return AlertDialog(
+            content: activeNotificationsDialogContent,
+            actions: [
+              FlatButton(
+                child: Text('OK'),
+                onPressed: () {
+                  Navigator.of(context).pop();
+                },
+              ),
+            ],
+          );
+        });
+  }
+
+  Future<Widget> _getActiveNotificationsDialogContent() async {
+    if (!Platform.isAndroid) {
+      return Text('"getActiveNotifications" is available only on Android');
+    }
+
+    DeviceInfoPlugin deviceInfo = DeviceInfoPlugin();
+    AndroidDeviceInfo androidInfo = await deviceInfo.androidInfo;
+    if (!(androidInfo.version.sdkInt >= 23)) {
+      return Text(
+        '"getActiveNotifications" is available only for Android API Level 23 and above',
+      );
+    }
+
+    try {
+      List<ActiveNotification> activeNotifications =
+          await flutterLocalNotificationsPlugin
+              .resolvePlatformSpecificImplementation<
+                  AndroidFlutterLocalNotificationsPlugin>()
+              ?.getActiveNotifications();
+
+      return Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          Text(
+            'Active Notifications',
+            style: TextStyle(fontWeight: FontWeight.bold),
+          ),
+          Divider(color: Colors.black),
+          if (activeNotifications.isEmpty) Text('No active notification'),
+          if (activeNotifications.isNotEmpty)
+            ...activeNotifications
+                .map(
+                  (ActiveNotification aN) => Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      Text(
+                        'id: ${aN.id.toString()}\n'
+                        'channelId: "${aN.channelId}"\n'
```

Comment:
I'd suggest removing the double quotation marks as it might confuse users into thinking they are part of the value itself. There isn't a need to distinguish if a value is a string when it's present in the app if that was the reason why they were placed there

Reply:
ok